core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3915,7 +3915,7 @@ public HttpRedirect doQuietDown(@QueryParameter boolean block, @QueryParameter i
      */
     @RequirePOST // TODO the cancel link needs to be updated accordingly
```

Comment:
```suggestion
    @RequirePOST
```

Almost certainly obsolete, or we'd know about it.